src/utils/os_zephyr.c

```diff
@@ -45,6 +45,8 @@ int os_get_reltime(struct os_reltime *t)
 	struct timespec ts;
 	int res;
 
+	clock_id = CLOCK_REALTIME;
```

Comment:
This should be a separate commit with appropriate commit log

Reply:
FYI, I have root cause and raised https://github.com/nrfconnect/sdk-hostap/pull/53 with proper fix (added your SOB too). So, no need to switch to REALTIME, we can get it working with any clock setting.